graphql/src/test/kotlin/com/trib3/graphql/execution/LeakyCauldronHooksTest.kt

```diff
@@ -233,7 +228,7 @@ class LeakyCauldronHooksTest {
     fun testOffsetDateTime() {
         val result = graphQL.execute("""query {offsetDateTime(o:"2019-10-30T00:01-07:00")}""")
             .getData<Map<String, String>>()
-        assertThat(result["offsetDateTime"]).isEqualTo("2019-10-31T07:01:00.000Z")
+        assertThat(result["offsetDateTime"]).isEqualTo("2019-10-31T00:01:00.000-07:00")
```

Comment:
I think this is because graphql-kotlin used to use jackson internally to translate between objects and strings and now does something different that results in more object re-use vs/ string reparsing (ie, the input object maintains its timezone which is now what gets returned -- that might be "better" anyway?).  the before/after asserted timestamps are the same instant which is what's important.